src/InputWithLabel.js

```diff
@@ -0,0 +1,38 @@
+
+import React from 'react';
+
+const InputWithLabel = ({
```

Comment:
This sort of looks like what was shown in the book. Let's just focus on what the directions suggest.